Gemfile.lock

```diff
@@ -1,75 +1,75 @@
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionmailbox (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      actionview (= 6.0.2.2)
-      activejob (= 6.0.2.2)
+    actionmailer (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      actionview (= 6.0.3.2)
+      activejob (= 6.0.3.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.2)
-      actionview (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionpack (6.0.3.2)
+      actionview (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
-      activestorage (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actiontext (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
+      activestorage (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
       nokogiri (>= 1.8.5)
-    actionview (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    actionview (6.0.3.2)
+      activesupport (= 6.0.3.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.2.2)
-      activesupport (= 6.0.2.2)
+    activejob (6.0.3.2)
+      activesupport (= 6.0.3.2)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activerecord (6.0.2.2)
-      activemodel (= 6.0.2.2)
-      activesupport (= 6.0.2.2)
-    activestorage (6.0.2.2)
-      actionpack (= 6.0.2.2)
-      activejob (= 6.0.2.2)
-      activerecord (= 6.0.2.2)
+    activemodel (6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activerecord (6.0.3.2)
+      activemodel (= 6.0.3.2)
+      activesupport (= 6.0.3.2)
+    activestorage (6.0.3.2)
+      actionpack (= 6.0.3.2)
+      activejob (= 6.0.3.2)
+      activerecord (= 6.0.3.2)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.2)
+    activesupport (6.0.3.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
-      zeitwerk (~> 2.2)
+      zeitwerk (~> 2.2, >= 2.2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.5)
+    autoprefixer-rails (9.8.6.1)
       execjs
     bindex (0.8.1)
-    bootsnap (1.4.6)
+    bootsnap (1.4.7)
       msgpack (~> 1.0)
     bootstrap (4.4.1)
       autoprefixer-rails (>= 9.1.0)
       popper_js (>= 1.14.3, < 2)
       sassc-rails (>= 2.0.0)
     builder (3.2.4)
-    byebug (11.1.1)
-    capybara (3.31.0)
+    byebug (11.1.3)
+    capybara (3.33.0)
```

Comment:
### [capybara](https://github.com/teamcapybara/capybara)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.31.0 → 3.33.0

[_[change-log](https://github.com/teamcapybara/capybara/blob/master/History.md), [source-code](https://github.com/teamcapybara/capybara)_]





<details>
<summary>Commits</summary>

A change of **86** commits. See the full changes on [the compare page](https://github.com/teamcapybara/capybara/compare/3.31.0...3.33.0).

These are the first 10 commits:
- (6c1d21b) [Fix getAttribute atom - issue #2300](https://github.com/teamcapybara/capybara/commit/6c1d21bd4c987ae1dfd18563c42e4e6dffbeeb8c)
- (8658b39) [Merge pull request #2301 from teamcapybara/issue_2300](https://github.com/teamcapybara/capybara/commit/8658b39d1c27956b2c85a3e6755d58fc99317537)
- (9905de4) [Fix results with negative max bounded ranges](https://github.com/teamcapybara/capybara/commit/9905de43f90d9739b2a35e9a040ac210e367d86d)
- (06f6c3d) [Merge pull request #2303 from teamcapybara/index_tests](https://github.com/teamcapybara/capybara/commit/06f6c3dbd6e20535e506c21c4268f3d4c492d6ae)
- (4d880c2) [Update History.md](https://github.com/teamcapybara/capybara/commit/4d880c237c6f0d8c907a6a6469233677ea3ea8a4)
- (2a7e1e9) [update Capybara::Minitest::Expectations documentation (#2305…](https://github.com/teamcapybara/capybara/commit/2a7e1e9684a883d8f50b8d42d60c7a874d430af4)
- (f05a2a0) [Update Capybara::Minitest::Assertion docs (#2310)](https://github.com/teamcapybara/capybara/commit/f05a2a033a958c4fa52889a309cec0829a57a62a)
- (6dc1615) [Query rewriting in rack_test form submission](https://github.com/teamcapybara/capybara/commit/6dc161550469380e173a6242434f0d360c81f3bd)
- (a7cd97e) [Style updates](https://github.com/teamcapybara/capybara/commit/a7cd97ed4d24e8cd4fc59b2349b639fca9fcb82b)
- (ec65805) [Merge pull request #2312 from teamcapybara/rack_form_submit](https://github.com/teamcapybara/capybara/commit/ec658052a08d340bdd74281261e1d224460b62c2)


</details>